distribution/docs/src/main/resources/content/_developing/_devComponents/managed-service-factories.adoc

```diff
@@ -142,14 +142,8 @@ See the documentation for https://docs.oracle.com/javase/7/docs/api/java/lang/Lo
 .Sample configuration file
 [source,linenums]
 ----
-authenticationTypes=[ \
-  "/\=", \
-  "/admin\=basic", \
-  "/system\=basic", \
-  "/sources\=basic", \
-  "/security-config\=basic", \
-  "/search\=basic", \
-  ]
+webAuthenticationTypes="basic"
+endpointAuthenticationTypes=""
```

Comment:
❓ What's the expected use case requiring different auth configurations for endpoints and web pages? I'm unclear why we need this distinction.

Reply:
I can think of cases where web pages need to be locked down by OIDC / Keycloak, but external clients need to authenticate via PKI or basic.

Reply:
In that case, it sounds like the distinction we need is not web pages vs. endpoints, but browser clients vs. non-browser clients. Is there actually a need for the former?

Reply:
Also, PKI isn't terminating, so in the case you described, you could apply `PKI|OIDC` to the whole system, and that would accommodate both types of clients. For BASIC, that wouldn't work, but I wonder if we could add a non-terminating basic handler (no prompt, just pulls credentials off the request). Then you could do `PKI|BASIC|OIDC`.

Just trying to make sure the config isn't more complicated than it needs to be

Reply:
Ok had an out-of-band discussion with @stustison. The conclusion was:
1. There's no way we could make a single auth type work for all the use cases we need to support. Our auth types are extensible, and that limits the assumptions we can make about how they will be used.
2. Making the configs browser clients vs. non-browser clients instead of web pages vs. endpoints would require changing the auth flow, which was not the intent of this issue. If we decide to do that, it will be in a separate PR.